packages/gasket-cli/test/unit/commands/create.test.js

```diff
@@ -25,116 +27,172 @@ mockActionStubs.installModules.update = jest.fn();
+    program.hook('preAction', (_, actionCommand) => { cmdOptions = actionCommand.opts(); });
+    return program;
+  })();
+
+  afterEach(() => {
     jest.clearAllMocks();
   });
 
-  it('executes expected bootstrap actions', async () => {
-    const cmd = new CreateCommand(['myapp', '--presets=nextjs']);
-    await cmd.run();
+  it('should have expected args', () => {
+    const args = cmd.commands[0]._args;
+    expect(CreateCommand.args.length).toEqual(args.length);
+    CreateCommand.args.forEach((arg, index) => {
+      expect(arg.name).toEqual(args[index]._name);
+      expect(arg.description).toEqual(args[index].description);
+      expect(arg.required).toEqual(args[index].required);
+    });
+  });
 
-    expect(mockActionStubs.mkDir).toHaveBeenCalled();
+  it('should have expected options', () => {
+    const options = cmd.commands[0].options;
+    expect(CreateCommand.options.length).toEqual(options.length);
+
+    CreateCommand.options.forEach((option, index) => {
+      if (!option.short) {
+        if (!options[index].long) expect(`--${option.name}`).toEqual(options[index].short);
+      }
+
+      if (option.short) {
+        expect(`--${option.name}`).toEqual(options[index].long);
+        expect(`-${option.short}`).toEqual(options[index].short);
+      }
+
+      if (option.default) expect(option.default).toEqual(options[index].defaultValue);
+      if (option.parse) expect(option.parse).toEqual(options[index].parseArg);
+      expect(option.description).toEqual(options[index].description);
+    });
+  });
+
+  it('allows for shorthand options', async () => {
+    await cmd.parseAsync(['node', 'gasket', 'create', 'myapp', '-p', 'nextjs,react']);
+    expect(cmdOptions.presets).toEqual(['nextjs', 'react']);
+  });
+
+  it('allows for longhand options', async () => {
+    await cmd.parseAsync(['node', 'gasket', 'create', 'myapp', '--presets', 'nextjs,react']);
+    expect(cmdOptions.presets).toEqual(['nextjs', 'react']);
+  });
+
+  it('executes expected bootstrap actions', async () => {
+    await cmd.parseAsync(['node', 'gasket', 'create', 'myapp']);
     expect(mockActionStubs.loadPreset).toHaveBeenCalled();
     expect(mockActionStubs.globalPrompts).toHaveBeenCalled();
+    expect(mockActionStubs.mkDir).toHaveBeenCalled();
     expect(mockActionStubs.setupPkg).toHaveBeenCalled();
     expect(mockActionStubs.writePkg).toHaveBeenCalled();
     expect(mockActionStubs.installModules).toHaveBeenCalled();
     expect(mockActionStubs.linkModules).toHaveBeenCalled();
-    expect(mockActionStubs.postCreateHooks).toHaveBeenCalled();
   });
 
-  it('skips bootstrap actions with --no-bootstrap', async () => {
-    const cmd = new CreateCommand(['myapp', '--no-bootstrap', '--presets=nextjs']);
-    await cmd.run();
-
+  it('skips bootstrap actions with --bootstrap', async () => {
+    await cmd.parseAsync(['node', 'gasket', 'create', 'myapp', '--bootstrap', '--presets=nextjs']);
+    expect(cmdOptions.bootstrap).toBe(true);
```

Comment:
This and other tests need adjust regarding previous `--no-*` option comment

```suggestion
  it('skips bootstrap actions with --no-bootstrap', async () => {
    await cmd.parseAsync(['node', 'gasket', 'create', 'myapp', '--no-bootstrap', '--presets=nextjs']);
    expect(cmdOptions.bootstrap).toBe(false);
```